examples/ios/Examples/OpenCloseRealm.swift

```diff
@@ -56,4 +55,19 @@ class OpenCloseRealm: AnonymouslyLoggedInTestCase {
         let realm = try! Realm(configuration: config)
         // :code-block-end:
     }
+
+    func testConfigureObjectTypes() {
+        // :code-block-start: configure-object-types
+        var config = Realm.Configuration.defaultConfiguration
+        // :remove-start:
+        config.inMemoryIdentifier = "test"
+        // :remove-end:
+
+        // Given: `class Task: Object`
+        // Limit the realm to only the Task object
```

Comment:
```suggestion
        // Limit the realm to only the Task object:
```
nitpick: for consistency with objective-c